deepstream/python_apps/deepstream-pravega-demos/pravega-to-object-detection-to-pravega.py

```diff
@@ -636,6 +709,11 @@ def main():
             # Always write to Pravega immediately regardless of PTS
             pravegasink.set_property("sync", False)
             pravegasink.set_property("timestamp-mode", "tai")
+            if args.health_check_enabled:
+                idle_detector = IdleDetector(args.health_check_idle_seconds)
+                HealthCheckHttpHandler.idle_detector = idle_detector
+                pravegasinkpad = pravegasink.get_static_pad("sink")
```

Comment:
You also need to add this probe to the msgbroker element.

Reply:
But we can't gurantee object detected in a specific period.

Reply:
Ok I see. In that case, put the probe on pravegasrc or some other common element. If anything goes wrong with the pipeline, pravegasrc will eventually block when the queues fill up.